src/main/java/com/hm/petmaster/listener/PlayerInteractListener.java

```diff
@@ -140,6 +134,17 @@ public void onPlayerInteractEntityEvent(PlayerInteractEntityEvent event) {
 		}
 	}
 
+	@EventHandler(ignoreCancelled = true)
+    public void onPlayerDamagePet(EntityDamageByEntityEvent event){
```

Comment:
The _PlayerInteractListener_ class is starting to get quite big, what do you think about adding the new functionality into a separate class? We could register it from the main _PetMaster_ class if and only if `disablePlayerDamage` is set to true, which would be more efficient as the listener would code would not be triggered otherwise. 😉